adminutils/adminutils.py

```diff
@@ -183,6 +183,62 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
         else:
             await ctx.tick()
 
+    @emoji.command(name="steal")
+    async def emote_steal(self, ctx, name: str, message_id: int, *roles: discord.Role):
+        """
+        Steal an emoji from a message ID and add it to your server.
+        Role lock the emoji if you wish (use double quotes if role name has spaces)
+
+        Examples:
+            `[p]emoji steal thisReallyCoolName 743291269471797248`
+            `[p]emoji steal thisReallyCoolName 743291269471797248 RoleName`
+        """
+        channel = ctx.channel
+        message = await channel.fetch_message(message_id)
+        dig_up_emoji_check = message.content.split(":")[-3]
+        check_animated = dig_up_emoji_check.endswith("<a")
+        dig_up_emoji = message.content.split(":")[-1]
+        emoji = "".join(x for x in dig_up_emoji if x.isdigit())
+        check_animated = dig_up_emoji_check.endswith("<a")
+        if check_animated is True:
+            format_image = "gif"
+        else:
+            format_image = "png"
+        url = f"https://cdn.discordapp.com/emojis/{emoji}.{format_image}"
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url) as r:
+                data = await r.read()
+        try:
+            await ctx.guild.create_custom_emoji(
+                name=name,
+                image=data,
+                roles=roles,
+                reason=get_audit_reason(
+                    ctx.author,
+                    ("Restricted to roles: {}").format(
+                        ", ".join([f"{role.name}" for role in roles])
+                    )
+                    if roles
+                    else None,
+                ),
+            )
+            await ctx.tick()
+            msg = "Emote added, fam!"
+            e = discord.Embed(title=msg, colour=discord.Colour.dark_green())
+            e.set_author(name=ctx.author.name, icon_url=ctx.author.avatar_url)
+            e.set_image(url=url)
+            e.set_thumbnail(url=ctx.guild.icon_url)
+
+            await ctx.send(embed=e)
+        except discord.InvalidArgument:
+            await ctx.send(
+                "Make sure there's an emoji in that messageID and you're running this command correctly (`[p]emoji steal nameTheEmote <msgID>`) Extra numbers in the message will throw this error too."
+            )
+            return
+        except discord.HTTPException:
+            await ctx.send("You are at your emoji limit it would seem.")
+            return
```

Comment:
```suggestion
        except discord.HTTPException as e:
            await ctx.send(chat.error(_("An error occurred on adding an emoji: {}").format(e)))
```
HTTPException can happen not only on hitting emoji limit.
About emoji limit - perhaps even better handle this at start of command via check for count of current server emojis and total available emojis (`guild.emoji_limit`)